search/binary_search_test.cpp

```diff
@@ -0,0 +1,30 @@
+/// Copyright 2020 @author Albirair
```

Comment:
could you also add
```c++
/**
  * @file
  *
  * A generic implementation of binary_search.
  */
```
to this file for consistency. I am building an document generator [here (repo)](https://github.com/kvedala/C-Plus-Plus) that creates the [online documentation](https://kvedala.github.io/C-Plus-Plus)

Reply:
I added the **file** Doxygen command to `search/binary_search_test.cpp` as suggested by @kvedala in this [latest commit.](https://github.com/TheAlgorithms/C-Plus-Plus/pull/738/commits/16fec03ac899cdc847f20e553de7850a83caaa71)